crates/solidity/testing/sourcify/src/events.rs

```diff
@@ -52,6 +56,10 @@ impl Events {
         let unresolved = reporter.add_counter("❔ Unresolved", Color::White, 0);
         let incompatible = reporter.add_counter("❕ Incompatible", Color::White, 0);
 
+        let definitions = reporter.add_counter("Definitions", Color::White, 0);
+        let references = reporter.add_counter("References", Color::White, 0);
+        let unresolved_references = reporter.add_counter("Unresolved refs", Color::White, 0);
```

Comment:
I wonder if we should dedupe the existing counters instead? I'm not sure how to distinguish between `failed`, `unresolved`, and `unresolved_references`.